packages/p2p/source/socket-server/server.test.ts

```diff
@@ -72,16 +72,17 @@ describe<{ sandbox: Sandbox; server: Server }>("Server", ({ it, assert, beforeEa
 		context.server = context.sandbox.app.resolve(ServerProxy);
 	});
 
-	it("#initialize - should instantiate a new Hapi server", async ({ server }) => {
-		const spyHapiServerRegister = spy(HapiServerMock.prototype, "register");
+	it.only("#initialize - should instantiate a new Hapi server", async ({ server }) => {
```

Comment:
remove .only

Reply:
We are skipping all tests in server.test.ts file. I will fix that in separete PR.